node_modules/oae-zencoder/lib/rest.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var OAE = require('oae-util/lib/oae');
+var OaeServer = require('oae-util/lib/server');
+var log = require('oae-logger').logger('zencoder-rest');
+
+var ZencoderAPI = require('./api');
+
+OaeServer.addSafePathPrefix('/api/zencoder/encodingNotification');
+
+/**
+ * @REST postZencoderNotification
+ *
+ * Send notification when encoding job is finished
+ *
+ * @Api         private
+ * @Server      tenant
+ * @Method      POST
+ * @Path        /zencoder/encodingNotification
+ * @BodyParam   {notification}    body         Upload metadata from Zencoder
+ * @Return      {void}
+ * @HttpResponse                  200          Encoding completed successfully
+ */
+OAE.tenantRouter.on('post', '/api/zencoder/encodingNotification', function(req, res) {
+    // Tell Zencoder we received the notification or they'll try to resend it
+    res.send(202, {message: 'Notification received'});
+
+    ZencoderAPI.encodingCompleted(req.body, function(err) {
+        if (err) {
+            log().error('Could not save video previews', err);
```

Comment:
After looking at the error handling in the API a bit more, if we assume the API will provide sufficient error information on the `err` object itself, then this line can just change to `log().error({'err': err}, 'Failed to save video previews');`